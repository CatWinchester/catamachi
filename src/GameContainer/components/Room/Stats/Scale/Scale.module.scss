.scale {
  display: inline-flex;
  flex: 1;
  flex-direction: column;
  .points {
    flex-wrap: wrap;
    display: inline-flex;
    flex-direction: row;
    .point {
      width:10px;
      height:10px;
      background: #f00;
      border-radius: 50%;
    }
  }
}
