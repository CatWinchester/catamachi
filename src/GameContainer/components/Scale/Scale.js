import React from 'react'
import PropTypes from 'prop-types'
import styles from './Scale.module.scss'

const Scale = (props) => {
  return <div className={styles.scale}>
    <h3>
      {props.type}
    </h3>
    {props.amount}%
  </div>
}

Scale.propTypes = {
  type: PropTypes.oneOf(['hunger', 'awakeness']).isRequired
}

export default Scale
