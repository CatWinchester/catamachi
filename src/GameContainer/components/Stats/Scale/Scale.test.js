import React from 'react';
import Scale from './Scale';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Scale type="hunger" amount={3}></Scale>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
