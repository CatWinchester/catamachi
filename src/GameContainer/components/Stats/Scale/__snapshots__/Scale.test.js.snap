// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className={undefined}
>
  <h2>
    hunger
  </h2>
  <h3>
    3
    %
  </h3>
  <div
    className={undefined}
  >
     
    <div
      className={undefined}
    />
    <div
      className={undefined}
    />
    <div
      className={undefined}
    />
     
  </div>
</div>
`;
